Content.Server/Holiday/Christmas/RandomGiftSystem.cs

```diff
@@ -60,7 +60,6 @@ private void OnUseInHand(EntityUid uid, RandomGiftComponent component, UseInHand
         var coords = Transform(args.User).Coordinates;
         var handsEnt = Spawn(component.SelectedEntity, coords);
         _adminLogger.Add(LogType.EntitySpawn, LogImpact.Low, $"{ToPrettyString(args.User)} used {ToPrettyString(uid)} which spawned {ToPrettyString(handsEnt)}");
-        EnsureComp<ItemComponent>(handsEnt); // For insane mode.
```

Comment:
I assume this line was making it possible to carry any gift in your hands or storage with you. If this wasn't causing too many bugs or problems for admins before, I would keep it, since it only affects insane mode items (non-insane items are already selected by having the ItemComponent). Or maybe make another tree prototype with unsafe, but without insane items, so admins won't have to handle this for a whole month.
In any case, a change like this should at least be mentioned in the PR description.

Reply:
I changed it for two reasons:
- It makes gift entities different than the normal world entity
- It leads to bizarre stuff since it ensures the default Item comp, so you get like corpse wearing bag that you can stuff in a survival box.